Smarty/templates/DetailView.tpl

```diff
@@ -342,14 +344,14 @@ clipcopyobject.on('error', function(e) { clipcopyclicked = false; });
 																			<span class="slds-assistive-text">{$customlink_label}</span>
 																			</span>
 																		{else}
-																			<a class="webMnu" href="{$customlink_href}">
```

Comment:
should we urlencode or addslashes or something similar to avoid javascript errors and security injections?

Reply:
I applied addslashes to avoid js errors. functions like urlencode or similiar we cannot apply here because in some use cases we are executing javascript functions in the actions, and these function will break the js function